opts/opts.go

```diff
@@ -203,6 +203,9 @@ func ValidatePath(val string) (string, error) {
 	}
 
 	splited := strings.SplitN(val, ":", 2)
+	if splited[0] == "" {
+		return val, fmt.Errorf("bad format for volumes: %s", val)
+	}
 	if len(splited) == 1 {
 		containerPath = splited[0]
 		val = path.Clean(splited[0])
```

Comment:
I was the one that added that comment, not because the func shouldn't validate the mode part, but rather to make sure people knew that it didn't.  I actually would prefer if it did. If you want to do that as part of a different PR that would be fine with me, but we probably need to do the other part I mentioned in the previous comment.